gamedata/alldefs_post.lua

```diff
@@ -87,7 +87,7 @@ function UnitDef_Post(name, uDef)
 		uDef.minCollisionSpeed = 2.5 -- define the minimum velocity(speed) required for all units to suffer fall/collision damage.
 		uDef.groundFrictionCoefficient = 0.01 --default 0.01 -- used to reduce a unit’s speed when skidding (moving in a direction other than heading)
 		uDef.rollingResistanceCoefficient = 0.05 --default 0.05 -- used to reduce a unit’s speed when exceeding their normal max speed at their current heading.
-		uDef.atmosphericDragCoefficient = 3 -- default unknown -- this slows units travelling through the air, likely due to explosion impulse launching them.
+		--uDef.atmosphericDragCoefficient = 3 -- default unknown -- this slows units travelling through the air, likely due to explosion impulse launching them.
```

Comment:
Delete commented out code

Reply:
this is barely documented anywhere, i want to keep it there with the rest of the global cluster

Reply:
If this is an undocumented tag, then it should be documented in the proper place, the engine docs, not in an obscure comment in a random file

Reply:
what about commented out right next to all the other constants that affect unit physics in the file "alldefs_post" that affects units in post?

Reply:
per our conversation over discord, I deleted the comment and the now unnecessary global changes to skidding